engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/blueprint/NadelExecutionBlueprintFactory.kt

```diff
@@ -342,12 +343,12 @@ private class Factory(
                     .filterIsInstance<AnyNamedNode>()
                     .filterNot {
                         it is ObjectTypeExtensionDefinition
-                            || it is InterfaceTypeExtensionDefinition
-                            || it is EnumTypeExtensionDefinition
-                            || it is ScalarTypeExtensionDefinition
-                            || it is InputObjectTypeExtensionDefinition
-                            || it is SchemaExtensionDefinition
-                            || it is UnionTypeExtensionDefinition
+                                || it is InterfaceTypeExtensionDefinition
```

Comment:
Have you run `./gradlew ktlintSetCodeStyle` in the project? Continued indentation should be 4 spaces not 8.

After running that, you may need to fix up the Java codestyle to not use `*` imports.

Reply:
is it `./gradlew ktlintFormat`?
it leaves this file as it is

Reply:
if you want to enforce a certain style, we should make the build break if the code does not adhere to that style

Reply:
`ktlintFormat` looks for the source files in the wrong place apparently. After fixing the path, it does apply style changes but it also removes some used imports which prevents code from compiling so I won't touch it for now